src/github/client/github-installation-client.ts

```diff
@@ -33,17 +37,19 @@ export class GitHubInstallationClient {
 	private readonly appTokenHolder: AppTokenHolder;
 	private readonly installationTokenCache: InstallationTokenCache;
 	public readonly githubInstallationId: InstallationId;
+	private readonly gitHubEnterprise: GitHubClientConfig | undefined;
 	private readonly logger: Logger;
 
 	constructor(
 		githubInstallationId: InstallationId,
 		logger: Logger,
+		gitHubEnterprise?: GitHubClientConfig | undefined,
```

Comment:
you don't have to do ` | undefined` if your param is already optional, that's implied

Reply:
that's because your `githubEnterprise` in the class is set to `GithubClientConfig` only and needs to be `GithubClientConfig | undefined`.  

And since we're not using the accept headers anymore, we could just remove this and use the `baseURL` we used to have that combines both the domain and the path (for the GHE case) which should work for both GHE and github.com :)